Dockerfile

```diff
@@ -1,24 +1,29 @@
-FROM golang:1.12-alpine
+FROM golang:1
 
 ENV PROJECT=coreos-version-checker
-COPY . /${PROJECT}-sources/
+ENV ORG_PATH="github.com/Financial-Times"
+ENV REPO_PATH="${ORG_PATH}/${PROJECT}"
+ENV SRC_FOLDER="${GOPATH}/src/${REPO_PATH}"
 
-RUN apk --no-cache --virtual .build-dependencies add git curl \
-  && ORG_PATH="github.com/Financial-Times" \
-  && REPO_PATH="${ORG_PATH}/${PROJECT}" \
+COPY . ${SRC_FOLDER}
+WORKDIR ${SRC_FOLDER}
+
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh \
+  && $GOPATH/bin/dep ensure -vendor-only \
   && mkdir -p $GOPATH/src/${ORG_PATH} \
   # Linking the project sources in the GOPATH folder
   && ln -s /${PROJECT}-sources $GOPATH/src/${REPO_PATH} \
-  && cd $GOPATH/src/${REPO_PATH} \
+  && cd ${SRC_FOLDER} \
   && echo "Fetching dependencies..." \
```

Comment:
Could you move this comment to the before fetching dep, or remove it altogether?